log.go

```diff
@@ -392,6 +392,7 @@ func (l *Logger) Log() *Event {
 // Arguments are handled in the manner of fmt.Print.
 func (l *Logger) Print(v ...interface{}) {
 	if e := l.Debug(); e.Enabled() {
+		e.skipFrame++
```

Comment:
When I said inline, I meant use the method on the same line as above, not actually inlining the function. Sorry for the confusion.

Reply:
That makes sense, I'll fix that up.

Would you object to putting it inline with the Msg call instead of the conditional line?